build/bin/sage-logger

```diff
@@ -63,21 +63,19 @@ fi
 
 timefile="$logdir/$logname.time"
 rm -f "$timefile"
-if /usr/bin/time -h -o /dev/null true 2>/dev/null; then
-    TIME="/usr/bin/time -h -o $timefile"
-else
-    TIME=""
-fi
+time_cmd() {
+    local max=$(($SECONDS+10))
+    exec 3>&1 4>&2
+    local out
+    TIME_OUTPUT=$((time 1>&3 2>&4 sh -c "$1") 2>&1)
+    local retstat=$?
```

Comment:
```suggestion
    local TIME_OUTPUT=$((time 1>&3 2>&4 sh -c "$1") 2>&1) retstat=$?
```
if you want't `TIME_OUTPUT` to be local, this is one way.

OTOH, if you keep `TIME_OUTPUT` global you can use it instead of `timefile`.